tests/smoke/test_dists.py

```diff
@@ -18,9 +18,9 @@
 
 
 @pytest.fixture(scope="function")
-def build():
-    sandbox.run_setup("setup.py", ["--quiet", "clean", "--all"])
-    return sandbox.run_setup("setup.py", ["--quiet", "sdist", "bdist_wheel"])
+def build() -> None:
+    sandbox.run_setup("setup.py", ["--quiet", "clean", "--all"])  # type: ignore[no-untyped-call]
+    sandbox.run_setup("setup.py", ["--quiet", "sdist", "bdist_wheel"])  # type: ignore[no-untyped-call]
```

Comment:
Not available in typeshed, but not worth it as setup.py-based builds already raise deprecation warnings.